examples/email-recovery/src/components/Recovery.tsx

```diff
@@ -0,0 +1,42 @@
+"use client";
+
+import { IframeStamper } from "@turnkey/iframe-stamper";
+import { Dispatch, SetStateAction, useEffect, useState } from "react";
+
+interface RecoveryProps {
+  iframeUrl: string;
+  turnkeyBaseUrl: string;
+  setIframeStamper: Dispatch<SetStateAction<IframeStamper | null>>;
+}
+
+const TurnkeyIframeContainerId = "turnkey-iframe-container-id";
+const TurnkeyIframeElementId = "turnkey-iframe-element-id";
+
+export function Recovery(props: RecoveryProps) {
+  const [iframeStamper, setIframeStamper] = useState<IframeStamper | null>(
+    null
+  );
+
+  useEffect(() => {
+    if (!iframeStamper) {
+      const iframeStamper = new IframeStamper({
+        iframeUrl: props.iframeUrl,
+        iframeContainerId: TurnkeyIframeContainerId,
+        iframeElementId: TurnkeyIframeElementId,
+      });
+      iframeStamper.init().then(function () {
```

Comment:
any reason not to use an fat arrow function here for consistency?

Reply:
no reason in particular. We could use an arrow function 👍